applications/consdb/secrets.yaml

```diff
@@ -4,14 +4,24 @@ consdb-password:
   copy:
     application: sasquatch
     key: consdb-password
+exposurelog-password:
+  description: >-
+    PostgreSQL password for the exposurelog user exposurelog database.
+  copy:
+    application: exposurelog
+    key: exposurelog_password
 oods-password:
   description: >-
     PostgreSQL password for the OODS user Butler database.
 lfa-password:
   description: >-
     LFA password
-exposurelog-password:
-  description: "Password for the TTS where we use exposurelog database."
   copy:
-    application: exposure-log
-    key: exposurelog_password
+    application: auxtel
+    key: aws-secret-access-key
```

Comment:
We're sure this is the read-only key, not the read-write one, correct?

Reply:
I am more sure that this is not the read only key, in auxtel/secrets.yaml it mentions that these are for writing to the lfa.

Reply:
So I should switch it back to lfa-password with no copy directive, and get the password from 1password to give to the admins to put in the vault?

Reply:
Yes, that is the read/write key. Do we have a read only key for the TTS LFA?

Reply:
In 1Pass I only find what looks to be the same as what auxtel has, based on the names 'aws-secret-access-key(-id)'